plugins/reporters/spdx/src/main/kotlin/Extensions.kt

```diff
@@ -171,6 +174,20 @@ internal fun Package.toSpdxPackage(
     )
 }
 
+private fun OrtResult.getVcsScanResult(id: Identifier): ScanResult? =
+    getScanResultsForId(id).firstOrNull { it.provenance is RepositoryProvenance }
+
+private fun OrtResult.getResolvedRevision(id: Identifier): String? =
+    (getVcsScanResult(id)?.provenance as RepositoryProvenance?)?.resolvedRevision
+
+private fun OrtResult.getPackageVerificationCode(id: Identifier, type: SpdxPackageType): String? =
+    when (type) {
+        SpdxPackageType.VCS_PACKAGE -> getVcsScanResult(id)?.summary?.packageVerificationCode?.takeUnless {
```

Comment:
What about `SpdxPackageType.PROJECT` here? Shouldn't we also be able to compute the package verification codes for projects?

Reply:
Yes, that's also possible. My intention was to, in the first place, only re-implement it for packages - to get back the functionality we had before. IIRC we haven't ever reported package verification codes for projects. If you're fine I'd add this later, in particular as I fear that for projects I've introduced a bug (mentioned in general slack by Nicola), which I need to investigate first. If you're ok, I'd leave this for now up to a following PR?

Reply:
> IIRC we haven't ever reported package verification codes for projects.

That's indeed the question, but looks like you're right: The original code had

    val spdxProjectPackage = project.toPackage().toSpdxPackage(SpdxPackageType.PROJECT, licenseInfoResolver)

which does not pass a third argument of type `ScanResult` to `toSpdxPackage()`, so it defaults to `null`, which ripples down the package verification code being `null` for projects.

> If you're fine I'd add this later

Sure, esp. as I've convinced myself now that it's new functionality.